db/employees-db.js

```diff
@@ -0,0 +1,21 @@
+const faker = require('faker')
+
+module.exports.generateEmployees = () => {
+    let employees = [];
+
+    for(let i = 0; i < 50; i++) {
+        let department = Math.floor((Math.random() * 20) + 1);
+        let firstName = faker.name.firstName();
+        let lastName = faker.name.lastName();
+        let hireDate = '2006-04-25T23:59:03.244Z';
+
+        employees.push({
+            "department": department,
+            "first_name": firstName,
+            "last_name": lastName,
+            "hire_date": hireDate
+        });
+    };
+    console.log(employees)
```

Comment:
console.log needs to be removed

Reply:
Runs and creates database with information populated, but throws npm error afterwards.  Can another person please pull it and test it while I update my version just in case.  Don't think this is an issue with J's code.